OneCommunityGlobal/HighestGoodNetworkApp
# Haohui - Gives the user permission to add new user permissions role

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/0576f59d-b327-4e88-b12f-3e94a47f662e)

## Related PRS (if any):
This frontend PR is related to the #546 backend PR.
…

## How to test:
1. check into current branch
2. do npm run build and npm run start to run this PR locally
3. log as owner user call user A
4. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions
5. input another user name, call user B
6. find the “Add New User Permissions Role”, click the add button
7. scroll down and click submit button
8. log out and login as user B(if already logged in, make sure to log out and log back in)
9. go to dashboard→ Other Links→ Permissions Management→ Add New Role
10. create new role, input the role name, select permissions, then submit
11. the new role show in the "user roles" table

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/f9e5d3bb-9392-422c-9ae7-9ba931ed6b8c




## Note:
Include the information the reviewers need to know.
